src/quo2/components/switcher/switcher_cards.cljs

```diff
@@ -1,27 +1,30 @@
 (ns quo2.components.switcher.switcher-cards
   (:require [react-native.core :as rn]
-            [status-im.i18n.i18n :as i18n]
+            [react-native.fast-image :as fast-image]
             [quo2.foundations.colors :as colors]
             [quo2.components.markdown.text :as text]
             [quo2.components.buttons.button :as button]
             [quo2.components.switcher.styles :as styles]
             [quo2.components.counter.counter :as counter]
             [quo2.components.tags.status-tags :as status-tags]
-            [status-im.ui.components.fast-image :as fast-image]
             [quo2.components.avatars.user-avatar :as user-avatar]
             [quo2.components.avatars.group-avatar :as group-avatar]
             [quo2.components.list-items.preview-list :as preview-list]
-            [quo2.components.avatars.channel-avatar :as channel-avatar]))
+            [quo2.components.avatars.channel-avatar :as channel-avatar]
+            [status-im.i18n.i18n :as i18n]))
+
+;;TODO move outside quo or find a way how to move i18n outside
```

Comment:
moving outside quo2 will break the Figma structure we are following for components?
`quo2/componenet_directory/component`